### PR TITLE
Update

### LINKED ISSUE #55: Update dependencies
Updating `babel-cli` will fix all security alerts, if my analysis is correct.

Update all dependencies if possible.

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Fixes #55

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Please extra focus on `body.js` I am not sure if I did everything correct there with `parameter` and `parameter_`

File: lib/body.js [hunk 1]
```diff
@@ -21,36 +21,36 @@ class Body {
 
     // Get the skills parameters from the parameter server to extract the
     // possible motions.
-    const param = new Param({
+    const parameter = new Param({
```
Comment (Rayman) [inline lib/body.js:24]: What is wrong with param? It's an instance of Param which is in the roslibjs api

File: lib/body.js [hunk 2]
```diff
@@ -21,36 +21,36 @@ class Body {
...
       ros,
       name: 'skills'
     });
 
     // Extract all relevant parameter from the parameter group
     const that = this;
-    param.get(param => {
+    parameter.get(parameter_ => {
```
Comment (Rayman) [inline lib/body.js:31]: `parameter_` is a really strange name, what's wrong with using the same name?

File: lib/body.js [hunk 3]
```diff
@@ -21,36 +21,36 @@ class Body {
...
       // Arm joint names. N.B.: the suffices 'left' and 'right'
       // must be added
       that.jointNames.arm = [];
       that.jointNames.leftArm = [];
       that.jointNames.rightArm = [];
-      param.arm.joint_names.forEach(e => {
-        that.jointNames.arm.push(`${e}`);
-        that.jointNames.leftArm.push(`${e}_left`);
-        that.jointNames.rightArm.push(`${e}_right`);
+      parameter_.arm.joint_names.forEach(each => {
```
Comment (Rayman) [inline lib/body.js:39]: Maybe a better name of each would be: `jointName`

File: lib/ed.js [hunk 4]
```diff
@@ -83,68 +83,68 @@ class Ed extends EventEmitter2 {
 
     for (const entity of entities) {
       const {id} = entity;
-      const newObj = {id};
+      const newObject = {id};
```
Comment (Rayman) [inline lib/ed.js:86]: You are correct that obj & msg are too short

File: lib/body.js (hunk 1 above)
Comment (MatthijsBurgh) [reply lib/body.js:24]: XO wants it updated

File: lib/body.js (hunk 2 above)
Comment (MatthijsBurgh) [reply lib/body.js:31]: XO wants it updated

File: lib/body.js (hunk 1 above)
Comment (Rayman) [reply lib/body.js:24]: Ok, fine 😆

**Pull request merged**